angularjs/Diagram/Getting-Started.md

```diff
@@ -74,7 +74,7 @@ Add a placeholder `div` element that can be used to initialize the diagram widge
 {% endhighlight %}
 
 This creates an empty diagram
-![](/angularjs/Diagram/Getting-Started_images/Getting-Started_img1.png)
+![Empty diagram.](/angularjs/Diagram/Getting-Started_images/Getting-Started_img1.png)
```

Comment:
You can change the alt text like as "Create a diagram in AngularJS", Please give meaningful term.